sw/device/silicon_creator/lib/drivers/BUILD

```diff
@@ -191,6 +190,7 @@ opentitan_functest(
         "//hw/top_earlgrey/sw/autogen:top_earlgrey",
         "//sw/device/lib:flash_ctrl",
         "//sw/device/lib/dif:aon_timer",
+        "//sw/device/lib/dif:flash_ctrl",
```

Comment:
@a-will : Just an FYI, looks like this test is using both the flash_ctrl.c and dif_flash_ctrl.c  (since I know you said you were in the process of removing `flash_ctrl.c`.